### DISCUSSION TITLE
help to increase version of affected package

### DISCUSSION THREAD

**Discussion by bolanosdev** [jscutlery/semver#758] [Q&A] [answered]
hi I'm creating a little mono repo of ui libraries.
and so far I have configured it to run lint/test of affected packages.
I wanted to also create a command to bump the version of the affected packages and I thought  the only thing I needed to do was to run  nx affected -t version 

but when I do nx says there are not affected packages
but if I do nx affected -t lint or nx affected -t test it does recognized my package as affected.
do I need to add a separate configuration file for the semver plugin

my repo is public so you can see the main and the head in the following link
https://github.com/bolanosdev/phoenix-ui/pull/5/files

thanks for the help

**Comment by edbzn** [comment, marked as answer]
Hi @bolanosdev, I think the issue is due to the absence of "version" target defined for the projects. You probably need to create a project.json file for all the releasable projects with a "version" target defined, or even better, you could use [the inference plugin API](https://nx.dev/extending-nx/recipes/project-graph-plugins) to dynamically add the "version" target to your releasable projects.

**Comment by edbzn** [comment]
I'm going to mark it as answered, reach me here if my suggestion doesn't work.
